scurl/cgurl.pyx

```diff
@@ -347,25 +351,42 @@ cpdef urljoin(base, url, bool allow_fragments=True):
     This function intends to replace urljoin from urllib,
     which uses Resolve function from class GURL of GURL chromium
     """
+    # fallback to the stdlib if allow_fragments and base are not presented
+    if not (allow_fragments and base):
```

Comment:
I like the early fallback, makes it clear when something is not supported. Regarding base, ``base=None`` and `base=''`` are equivalent, I wonder if ``base=''`` will work with GURL? But that's not related to the PR.

Reply:
hey @lopuhin , I don't think empty string is supported by GURL 😄 I just made a continued PR just for that